lib/js/jsbigints.nim

```diff
@@ -0,0 +1,132 @@
+## Arbitrary precision integers.
```

Comment:
new modules should go under `std/`
yes, I know there's a tension with fact that this also belongs inside lib/js, but IMO the scope pollution rule is more important, and also the module starts with js (jsbingints) anyways.

(also applies to your other PR's)

Reply:
I dont know whatever folder as long as can be imported, make a decision about that and move it or not,
same with object/ref object, I dont know if the CI is prepared for JS modules on arbitrary folders tho.

Reply:
``std / jsbigints`` seems preferable indeed.